libs/types-common/src/Data/Id.hs

```diff
@@ -250,12 +250,14 @@ newtype RequestId = RequestId
                , NFData
                )
 
-instance Monoid RequestId where
-    mempty = RequestId "N/A"
-    mappend a b
+instance Semigroup RequestId where
+    (<>) a b
         | a == mempty = b
         | otherwise   = a
 
+instance Monoid RequestId where
+    mempty = RequestId "N/A"
```

Comment:
I guess this is fine as long as we're only using this for logging?  Perhaps we should add a comment about the broken Monoid instance to the newtype?

Reply:
It doesn't seem broken (re/ laws), just weird. I'll make a PR about replacing it with `instance Default RequestId`.

(I'd prefer if it was banished completely and we had a `Maybe RequestId`, but it'd be a more invasive change and this way we at least get rid of a weird instance.)